src/jobservice/job/impl/context.go

```diff
@@ -95,7 +96,14 @@ func (c *Context) Init() error {
 
 	db := getDBFromConfig(configs)
 
-	return dao.InitDatabase(db)
+	err = dao.InitDatabase(db)
+	if err != nil {
+		return err
+	}
+
+	// Initialize DB finished
+	initDBCompleted()
```

Comment:
Is it necessary?  IMO we can assume the DB is ready when DBSweeper sweeps the log?

Reply:
actually not, DB initialized after logger and DB initialization relay on logger.
Sweeper will be executed once as soon as logger initialization finished.